app/controllers/doorkeeper/openid_connect/discovery_controller.rb

```diff
@@ -30,6 +30,7 @@ def provider_response
           introspection_endpoint: oauth_introspect_url(protocol: protocol),
           userinfo_endpoint: oauth_userinfo_url(protocol: protocol),
           jwks_uri: oauth_discovery_keys_url(protocol: protocol),
+          end_session_endpoint: Doorkeeper::OpenidConnect.configuration.end_session_endpoint.call(),
```

Comment:
Could we please remove `()`after the `#call` ?

Reply:
Also you could reuse `openid_connect` instead of `::Doorkeeper::OpenidConnect.configuration` (see above lines)

Reply:
Will do myself

Reply:
Thank you so much! ✌️ 

Thank you for contributing so much to open source! 👍